tests/furusiya/model/components/test_fighter.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import Mock
+
+import pytest
+
+from model.components.fighter import Fighter
+from model.systems.fighter_system import FighterSystem
+
+
+@pytest.fixture
+def fighter():
+    yield Fighter(Mock(), 30, 5, 5, death_function=Mock())
+
+
+@pytest.fixture
+def bushslime():
+    yield Mock()
+
+
+@pytest.fixture
+def bushslime_fighter(bushslime):
+    bush_fighter = Fighter(bushslime, 15, 2, 2, death_function=Mock())
+    FighterSystem.set_fighter(bushslime, bush_fighter)
+    yield bush_fighter
+
+
+def test_take_damage(fighter):
```

Comment:
Please try to make your test methods ... sentences that describe functionality. For example, I would call this `test_take_damage_decreases_health`